snuba/settings_base.py

```diff
@@ -56,8 +56,8 @@
 
 DEFAULT_MAX_BATCH_SIZE = 50000
 DEFAULT_MAX_BATCH_TIME_MS = 2 * 1000
-DEFAULT_QUEUED_MAX_MESSAGE_KBYTES = 10000
-DEFAULT_QUEUED_MIN_MESSAGES = 10000
```

Comment:
The librdkafka defaults for these are 1048576 and 100000, respectively. They are documented [here](https://github.com/edenhill/librdkafka/blob/master/CONFIGURATION.md) as `queued.max.messages.kbytes` and `queued.min.messages`.